cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -217,6 +217,18 @@ class alignas(16) column_device_view_base {
     return null_mask()[word_index];
   }
 
+  __device__ bitmask_type get_mask_offset_word(size_type element_index) const noexcept
```

Comment:
I'm not seeing where this is being used in this PR.

Reply:
I think this can be moved to the ORC refactor PR.